src/0_storage.js

```diff
@@ -10,7 +10,7 @@ goog.require('utils');
 
 var COOKIE_DAYS = 365;
 
-var BRANCH_KEY_PREFIX = 'BRANCH_WEBSDK_KEY';
```

Comment:
Cool. This is the area I can help fix without you guys.

Basically I think you're trying to store two separate keys. One for the first time a session is detected. `branch_data` and the regular `branch_session` so I'm struggling to move forward without understanding what you guys were intending to check for the first session stored in cordova